tmexp/train_artm.py

```diff
@@ -203,12 +210,15 @@ def train_artm(
     )
 
     logger.info("Finished training.")
-    doctopic, _, _ = model_artm.get_theta_sparse()
+    doctopic, _, doc_indexes = model_artm.get_theta_sparse()
```

Comment:
could you add a TODO here ? this should be temporary, we should investigate why documents are removed and remove this at some point ?